editor/components/FatalityVictimsCard.tsx

```diff
@@ -0,0 +1,95 @@
+                  <div className="d-flex w-100 justify-content-start align-items-center">
+                    <h5 className="mb-1 me-2">
+                      <span className="fw-bold"> Unit {unit.unit_nbr}</span>
+                    </h5>
+                    <div className="d-flex flex-grow-1 justify-content-start text-secondary">
+                      <span>
+                        {unit.unit_desc?.label}
+                        {unitYearMakeModel ? ` | ${unitYearMakeModel}` : ""}
+                      </span>
+                    </div>
+                  </div>
+                  {unitVictims?.map((victim) => (
+                    <ListGroupItem key={victim.id} style={{ border: "none" }}>
+                      <div className="d-flex w-100 justify-content-between">
+                        <div className="d-flex align-items-center">
+                          <span className="fw-bold me-2">
+                            {victim.prsn_first_name} {victim.prsn_mid_name}{" "}
+                            {victim.prsn_last_name}
+                          </span>
+                          {victim.prsn_type_id !== 3 && // pedalcyclist
+                            // Dont show person type for cyclists or pedestrians bc its redundant
+                            victim.prsn_type_id !== 4 && ( // pedestrian
+                              <small className="text-secondary">
+                                {victim.prsn_type.label}
+                              </small>
+                            )}
+                        </div>
+                        {victim.injry_sev?.label && (
+                          <span
+                            className={`${getInjuryColorClass(victim.injry_sev.label)} d-inline-flex align-items-center justify-content-center px-2 py-1 rounded `}
+                            style={{ minWidth: "fit-content" }}
+                          >
+                            <small>{victim.injry_sev?.label}</small>
+                          </span>
+                        )}
+                      </div>
+                      <div className="mb-1 d-flex w-100 flex-column">
+                        <span className="mb-2">
+                          {victim.prsn_age} YEARS OLD -{" "}
+                          {victim.drvr_ethncty?.label} {victim.gndr?.label}
+                        </span>
+                        {victim.rest?.label &&
+                          // Only show restraint field for cars
+                          unit.unit_desc_id === 1 && // motor vehicle
+                          unit.veh_body_styl_id !== 71 && // motorycle
+                          unit.veh_body_styl_id !== 90 && ( // police motorcycle
+                            <span>Restraint used: {victim.rest.label}</span>
+                          )}
+                        {victim.prsn_exp_homelessness && (
+                          <span>{victim.prsn_exp_homelessness}</span>
+                        )}
```

Comment:
`prsn_exp_homelessness` is a boolean so this isn't rendering anything when it's true

Reply:
thank you!